src/components/Details/VmTemplateDetails/VmTemplateDetails.js

```diff
@@ -218,9 +212,8 @@ export class VmTemplateDetails extends React.Component {
                       updating={this.state.updating}
                       LoadingComponent={LoadingComponent}
                       onFormChange={(newValue, key, valid) => this.onFormChange('flavor', newValue, key, valid)}
-                      retrieveVmTemplate={() => retrieveVmTemplate(k8sGet, vmTemplate)}
                       formValues={settingsValue(this.state.form, FLAVOR_KEY)}
-                      onLoadError={this.onLoadError}
+                      template={vmTemplate}
```

Comment:
this component should still retrieve the base template for flavor options

Reply:
@suomiy , can we have a recursion of template depending on another template ?

will re-add loading of base template, just to be sure, but not sure if we need to, WDYT ?

Reply:
not real recursion (just the recursion of types (: ) But yes, user vm templates are depending on pre-instaled templates (that's where we get our flavors from).

Reply:
:+1:  Thanks !